library/gradle

```diff
@@ -6,92 +6,122 @@ GitRepo: https://github.com/keeganwitt/docker-gradle.git
-GitCommit: 1d17a9a8ffeac966db523f07dc8fa79dfc30a0f9
+GitCommit: d3b56becf2affe337173f9c88805f2ab687027f6
 Directory: jdk8
 
 Tags: 8.10.2-jdk8-focal, 8.10-jdk8-focal, 8-jdk8-focal, jdk8-focal
 Architectures: amd64, arm32v7, arm64v8, ppc64le
-GitCommit: 1d17a9a8ffeac966db523f07dc8fa79dfc30a0f9
+GitCommit: d3b56becf2affe337173f9c88805f2ab687027f6
 Directory: jdk8-focal
 
+Tags: 8.10.2-jdk8-corretto, 8.10-jdk8-corretto, 8-jdk8-corretto, jdk8-corretto, 8.10.2-jdk8-corretto-al2023, 8.10-jdk8-corretto-al2023, 8-jdk8-corretto-al2023, jdk8-corretto-al2023
+Architectures: amd64, arm64v8
+GitCommit: d3b56becf2affe337173f9c88805f2ab687027f6
+Directory: jdk8-corretto
+
 Tags: 8.10.2-jdk11, 8.10-jdk11, 8-jdk11, jdk11, 8.10.2-jdk11-jammy, 8.10-jdk11-jammy, 8-jdk11-jammy, jdk11-jammy
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
-GitCommit: 1d17a9a8ffeac966db523f07dc8fa79dfc30a0f9
+GitCommit: d3b56becf2affe337173f9c88805f2ab687027f6
 Directory: jdk11
 
 Tags: 8.10.2-jdk11-focal, 8.10-jdk11-focal, 8-jdk11-focal, jdk11-focal
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
-GitCommit: 1d17a9a8ffeac966db523f07dc8fa79dfc30a0f9
+GitCommit: d3b56becf2affe337173f9c88805f2ab687027f6
 Directory: jdk11-focal
 
 Tags: 8.10.2-jdk11-alpine, 8.10-jdk11-alpine, 8-jdk11-alpine, jdk11-alpine
 Architectures: amd64
-GitCommit: 1d17a9a8ffeac966db523f07dc8fa79dfc30a0f9
+GitCommit: d3b56becf2affe337173f9c88805f2ab687027f6
 Directory: jdk11-alpine
 
+Tags: 8.10.2-jdk11-corretto, 8.10-jdk11-corretto, 8-jdk11-corretto, jdk11-corretto, 8.10.2-jdk11-corretto-al2023, 8.10-jdk11-corretto-al2023, 8-jdk11-corretto-al2023, jdk11-corretto-al2023
+Architectures: amd64, arm64v8
+GitCommit: d3b56becf2affe337173f9c88805f2ab687027f6
+Directory: jdk11-corretto
+
 Tags: 8.10.2-jdk17, 8.10-jdk17, 8-jdk17, jdk17, 8.10.2-jdk17-jammy, 8.10-jdk17-jammy, 8-jdk17-jammy, jdk17-jammy
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
-GitCommit: 1d17a9a8ffeac966db523f07dc8fa79dfc30a0f9
+GitCommit: d3b56becf2affe337173f9c88805f2ab687027f6
 Directory: jdk17
 
 Tags: 8.10.2-jdk17-focal, 8.10-jdk17-focal, 8-jdk17-focal, jdk17-focal, 8.10.2-jdk-focal, 8.10-jdk-focal, 8-jdk-focal, jdk-focal, 8.10.2-focal, 8.10-focal, 8-focal, focal
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
-GitCommit: 1d17a9a8ffeac966db523f07dc8fa79dfc30a0f9
+GitCommit: d3b56becf2affe337173f9c88805f2ab687027f6
 Directory: jdk17-focal
 
 Tags: 8.10.2-jdk17-alpine, 8.10-jdk17-alpine, 8-jdk17-alpine, jdk17-alpine, 8.10.2-jdk-alpine, 8.10-jdk-alpine
 Architectures: amd64
-GitCommit: 1d17a9a8ffeac966db523f07dc8fa79dfc30a0f9
+GitCommit: d3b56becf2affe337173f9c88805f2ab687027f6
 Directory: jdk17-alpine
 
+Tags: 8.10.2-jdk17-corretto, 8.10-jdk17-corretto, 8-jdk17-corretto, jdk17-corretto, 8.10.2-jdk17-corretto-al2023, 8.10-jdk17-corretto-al2023, 8-jdk17-corretto-al2023, jdk17-corretto-al2023
+Architectures: amd64, arm64v8
+GitCommit: d3b56becf2affe337173f9c88805f2ab687027f6
+Directory: jdk11-corretto
```

Comment:
@keeganwitt 
Could it be that the Directory: jdk11-corretto should actually be Directory: jdk17-corretto?

Could you kindly confirm when you have a moment?
Thank you!

Reply:
Good catch! This was a mistake. #17881 opened to fix this.

Reply:
Thank you for addressing the feedback! I appreciate the quick updates.